tests/test_optimizer.py

```diff
@@ -1136,15 +1136,16 @@ def test_proximal_jacobian():
         deriv_mode="batched",
         use_jit=False,
     )
-    obj2 = ObjectiveFunction(
-        (
-            QuasisymmetryTripleProduct(eq2, deriv_mode="fwd"),
-            AspectRatio(eq2, deriv_mode="fwd"),
-            Volume(eq2, deriv_mode="fwd"),
-        ),
-        deriv_mode="looped",
-        use_jit=False,
-    )
+    with pytest.warns(DeprecationWarning, match="looped"):
```

Comment:
The deprecated `deriv_mode="looped"` is tested both here and in `test_objective_funs.py`, and the test coverage for this PR is pretty low. I recommend changing one of these two tests to another case to increase coverage.